hardhat.config.zksync.ts

```diff
@@ -147,7 +147,7 @@ const config: HardhatUserConfig = {
 };
 
 function isFork() {
-  return process.env.FORK === "true"
+  return process.env.FORKED_NETWORK
```

Comment:
```suggestion
  return !!process.env.FORKED_NETWORK
```

Reply:
I had this but the linter was giving errors and since something that is truthy (a nonempty string) works I didn't dig deeper into the lint rules